app/models/manageiq/providers/amazon/storage_manager/s3/cloud_object_store_container.rb

```diff
@@ -0,0 +1,26 @@
+class ManageIQ::Providers::Amazon::StorageManager::S3::CloudObjectStoreContainer < ::CloudObjectStoreContainer
+  include_concern 'Operations'
+
+  supports :delete do
+    unless ext_management_system
+      unsupported_reason_add(
+        :delete,
+        "CloudObjectStoreContainer has no #{ui_lookup(:table => "ext_management_systems")}, unable to delete it"
+      )
+    end
+  end
+
+  supports :empty do
+    unless ext_management_system
+      unsupported_reason_add(
+        :empty,
+        "CloudObjectStoreContainer has no #{ui_lookup(:table => "ext_management_systems")}, unable to empty it"
+      )
+    end
+  end
```

Comment:
@miha-plesko - is this the only reason ```empty``` would be unsupported? If the bucket is empty should the menu item in the UI still be enabled?